intellij-plugin-structure/structure-ide/src/main/java/com/jetbrains/plugin/structure/ide/IntelliJPlatformProduct.kt

```diff
@@ -34,7 +34,9 @@ enum class IntelliJPlatformProduct(
   ANDROID_STUDIO("AI", "Android Studio", "AndroidStudio"),
   DATA_SPELL("DS", "DataSpell", "DataSpell"),
   JETBRAINS_CLIENT("JBC", "JetBrains Client", "JetBrainsClient"),
-  GATEWAY("GW", "Gateway", "Gateway");
+  GATEWAY("GW", "Gateway", "Gateway"),
+  FLEET("FL", "Fleet", "Fleet"),
```

Comment:
Will verification against Fleet be the same as for other ij-platform products? I'm not sure that Fleet should be here.

Reply:
This is not actually used in the scope of this PR. It's just to make the list up-to-date. (See _aqua_, too).